### DISCUSSION TITLE
新增网页剪切功能

### DISCUSSION THREAD

**Discussion by gclm** [tw93/MiaoYan#245] [Ideas]
目前试用一段时间了。目前发现可以完全替代typora了。但是希望可以新增网页剪切功能。毕竟对于有部分网上技术文档写的真心不错。如果能有剪切功能的话。就可以本地保存了。而不是保存个链接。如果什么时候链接失效了，你收藏的文档就没了

## 参考链接： 
- https://github.com/webclipper/web-clipper

**Comment by tw93** [comment]
我去调研调研，的确有这个保存文章的诉求，不过可能需要新加一种文档模式了
